app/components/Feed/(routes)/Home/page.tsx

```diff
@@ -0,0 +1,125 @@
+"use client";
+
+import { dot, evidencForm, eye, messages, saved, users } from "@/app/assets/icons";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/navigation";
+import { Post } from "../../dummydata";
+import Search from "../../sectiions/search/header";
+
+
+export default function Home() {
+  const router = useRouter();
+
+  return (
+    <div className=" w-5/6  md:pt-5 pr-2   ">
+      <Search placeholder={""} />
+      <div className=" my-8 grid grid-cols-1 md:grid-cols-2 gap-4 md:gap-10 align-center md:justify-center md:align-center 
+">
+        {Post.map((item) => (
+          <div key={item.id} className="flex pe-1 flex-col justify-start items-start shrink-0 border-b border-gray-400 pb-5 ">
+            <div className="flex justify-between items-center gap-4  py-2 align-center  w-[100%]">
+              <div className="flex justify-between items-center gap-2   ">
+                <Image 
+                src={item.creatorImage} 
+                alt={item.creator} 
+                width={25} 
+                height={25} 
+                className=" rounded-[50%]" />
+                <p className="color-gray-700 text-center font-dm-sans text-base font-normal font-bold leading-6">
+                  <strong>{item.creator} <span className="text-slate-500 font-dmSans text-sm font-normal leading-5">. 5h</span></strong></p>
+              </div>
+              <div>
+                <Image src={dot} className="cursor-pointer" alt="more"/>
+              </div>
+            </div>
+
+           
+            <div className="w-[100%] relative ">
+              <Image
+               src={item.art} 
+               alt={item.description} 
+               className="w-[100%] h-447 shrink-0 rounded-md " />
+               <div 
+               className="absolute top-[10px] right-[20px] w-[35px] rounded-md bg-info bg-white/25 backdrop-blur-sm  flex justify-center p-2  cursor-pointer transition duration-500 ease-in-out">
+               <Image
+                src={item.artdescription}
```

Comment:
I know we had agreed on this routing but please remove Feed from components.

Let the feed folder be directly within the app folder

Then we do not need Feed/Home at all, since the Feed page is always the home page
You can move the content of Home/Page.tsx to Feed/Page.tsx and then get rid of the Home folder totally